lib/openshift/cluster_operator.rb

```diff
@@ -0,0 +1,22 @@
+module BushSlicer
+  class ClusterOperator < ClusterResource
+    RESOURCE = "clusteroperators"
```

Comment:
maybe we need a way to specify namespace of this resource, e.g. "k8s.io" but it is goot for the time being, just a thought